packages/back-end/package.json

```diff
@@ -13,7 +13,7 @@
     "dev": "yarn build:clean && concurrently \"yarn watch-compile\" \"yarn watch-dev | yarn pino-pretty\"",
     "build": "yarn build:clean && yarn build:typescript && yarn build:emails",
     "start": "node dist/server.js",
-    "start:with-tracing": "node --require @opentelemetry/auto-instrumentations-node/register dist/server.js",
+    "start:with-tracing": "node --require dist/tracing.js dist/server.js",
```

Comment:
Is there a good reason we need to change this?  If not, i would revert `package.json` and `yarn.lock`.  Will make the PR much less risky